src/components/Camera.tsx

```diff
@@ -0,0 +1,62 @@
+import { useRef, useEffect, useState } from "react";
+import "./Camera.css";
+
+export const Camera = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [isEnabled, setIsEnabled] = useState(false);
+  const videoRef = useRef<HTMLVideoElement | null>(null);
+
+  useEffect(() => {
+    if (!isEnabled || !videoRef.current) return;
```

Comment:
To make camera screen black srcObject should be set to null when !isEnabled
videoRef.current.srcObject = null;